pages/apis/graphql/graphql_cookbook.md

```diff
@@ -500,6 +500,21 @@ query GetJobRunTimeByBuild{
   }
 }
 ```
+### Cancel Job
+
+To cancel a job using its ID
+
+```
+mutation CancelJob{
+  jobTypeCommandCancel(input: { 
+    id: "{insert-job-id}"
+  }) {      
+    jobTypeCommand {
+      id
+    }
+  }
+}
```

Comment:
A couple of tweaks:

- Used **Prettify** from the GraphQL explorer.
- Matched input with other commands. Note, this removed the braces, so add them back if needed.

```suggestion
mutation CancelJob {
  jobTypeCommandCancel(input: { id: "job-id" }) {
    jobTypeCommand {
      id
    }
  }
}
```